examples/pylint/W1401_anomalous_backslash_in_string.py

```diff
@@ -1 +1,2 @@
-print('\d{3}')  # Error on this line
+import re
+re.findall('(\d+)\D*', '123 Main St.')  # Error on this line
```

Comment:
I don't like this example. Regular expressions are not typically covered in CSC108/148. Just do something even simpler than the original example, like `'This is a bad escape: \h'`.